packages/web-components/src/components/pictogram-item/__stories__/pictogram-item.stories.react.tsx

```diff
@@ -0,0 +1,213 @@
+	c-0.139,0.054-0.298,0.015-0.397-0.099c-0.03-0.033-2.983-3.368-4.158-4.271c-0.925-0.709-2.589-0.673-3.562-0.192
+	c-0.413,0.203-0.65,0.463-0.65,0.711c0.057,0.274,1.063,1.38,1.603,1.975L6.465,21.516z M10.755,11.729
+	C9.407,10.535,8.634,8.811,8.634,7c0-3.507,2.853-6.36,6.36-6.36s6.36,2.853,6.36,6.36c0,1.811-0.773,3.534-2.121,4.729
+	l-0.479-0.539c1.194-1.058,1.879-2.585,1.879-4.19c0-3.11-2.529-5.64-5.64-5.64c-3.11,0-5.64,2.53-5.64,5.64
+	c0,1.605,0.685,3.133,1.879,4.19L10.755,11.729z"
+    />
+    <rect id="_Transparent_Rectangle" style={{ fill: 'none' }} width="32" height="32" />
+  </svg>
+);
+/* eslint-enable max-len */
+
+/**
+ * Returns the pictogram based on the storybook knob value
+ *
+ * @param {string} sel string that defines the returning pictogram
+ * @returns {*} Pictogram SVG markup
+ */
+const selectPictogram = sel => {
+  switch (sel) {
+    case 'Desktop':
+      return Desktop;
+    case 'Pattern':
+      return Pattern;
+    case 'Touch':
+      return Touch;
+    default:
+      return '';
+  }
+};
+
+const pictograms = {
+  Desktop: 'Desktop',
+  Touch: 'Touch',
+  Pattern: 'Pattern',
+};
+
+const pictogramColors = {
+  'Text color (default)': COLOR_OPTIONS.DEFAULT,
+  Blue: COLOR_OPTIONS.BLUE,
+};
+
+export const Default = ({ parameters }) => {
+  const { heading, copy, href, linkCopy, pictogram, pictogramColor } = parameters?.props?.PictogramItem ?? {};
+  return (
+    <DDSPictogramItem color={pictogramColor}>
+      {pictogram?.src}
+      <DDSContentItemHeading>{heading}</DDSContentItemHeading>
+      <DDSContentItemCopy>{copy}</DDSContentItemCopy>
+      <DDSLinkWithIcon href={href} slot="footer">
+        {linkCopy} <ArrowRight20 slot="icon"></ArrowRight20>
+      </DDSLinkWithIcon>
+    </DDSPictogramItem>
+  );
+};
+
+Default.story = {
+  parameters: {
+    knobs: {
+      PictogramItem: ({ groupId }) => ({
+        heading: textNullable('Heading (heading):', 'Lorem ipsum dolor sit', groupId),
+        copy:
+          'Lorem ipsum dolor sit amet, ' +
+          'consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
+          'Ut enim ad minim veniam\n',
+        href: textNullable('Link with Icon href:', 'https://example.com', groupId),
+        linkCopy: textNullable('Link with Icon copy:', 'Lorem ipsum dolor', groupId),
+        pictogram: {
+          src: selectPictogram(select('Pictogram (required)', pictograms, pictograms.Desktop, groupId)),
+          'aria-label': textNullable('Aria-label:', 'Pictogram description', groupId),
+        },
+        pictogramColor: select('Pictogram color:', pictogramColors, COLOR_OPTIONS.DEFAULT, groupId),
+      }),
+    },
+  },
+};
+
+export default {
+  title: 'Components/Pictogram item',
+  decorators: [
+    story => {
+      return (
+        <div className="bx--grid">
+          <div className="bx--row">
+            <div className="bx--col-sm-16 bx--col-md-6">{story()}</div>
```

Comment:
```suggestion
            <div className="bx--col-sm-4 bx--col-lg-8 bx--no-gutter">{story()}</div>
```

The grid is off compared to the Web Components version of the Story, but this should fix it. Otherwise looks okay!